### DISCUSSION TITLE
[FR] Factures récurentes

### DISCUSSION THREAD

**Discussion by FredVisualOffice** [baimard/gestion#211] [Ideas]
Bonjour,
ce serait sympa de pouvoir programmer des facturations récurentes avec le choix de la récurence (tous les jours, semaines, mois, années...) avec une date d'émission de la facture ( par exemple tous les 15 du mois, tous les mois de janvier...)

**Comment by baimard** [comment]
Bonjour,

C'est vraiment une bonne idée. Je l’intègre dans la ROADMAP. Je ne sais pas du tout comment faire par contre :)
